assets/css/v2/lcd_common/departures/section.scss

```diff
@@ -11,4 +11,9 @@
     background-color: colors.$warm-neutral-80;
     border-radius: 2px;
   }
+
+  & > .departure-row.direction-split::after {
+    left: 0;
+    width: 100%;
```

Comment:
This should also be using the darker-gray color that's used to divide sections, rather than the lighter gray used between departure rows normally.